Berksfile

```diff
@@ -1,5 +1,5 @@
 source 'https://supermarket.chef.io'
 
-cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
+cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall', branch: 'ramereth/nfs-cleanup'
```

Comment:
So once the [firewall PR](https://github.com/osuosl-cookbooks/firewall/pull/120) gets merged this won't be needed anymore right?